addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -483,3 +483,17 @@ img.o_we_custom_image.mx-auto {
     flex: 1 1 auto;
     outline: none;
 }
+
+// Color picker dialog for JW Editor
+.o_colorpicker_container{
+    background-color: #000;
+    position: absolute;
+    top: 45px;
+    left: 45px;
+    z-index: 100;
+    width: 230px;
+    .btn, .o_color_picker_inputs{
+        color: white;
+        background-color: #444;
+    }
+}
```

Comment:
Notice the red sign. Missing empty line at end of file.

Reply:
My Nemesis...